.github/workflows/test.yml

```diff
@@ -11,7 +11,10 @@ jobs:
   build_and_test:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: 
+          - ubuntu-latest
+          - macos-15-large # x86_64
```

Comment:
it is a bit confusing, but it is indeed how GH Actions define arm64 and x86_64 images :)